cmd/keeper/keeper.go

```diff
@@ -1379,6 +1453,11 @@ func (p *PostgresKeeper) postgresKeeperSM(pctx context.Context) {
 			return
 		}
 
+		if err = p.recreateAddonReplicationSlots(db.Spec.AdditionalReplicationSlotNames); err != nil {
+			log.Errorw("error updating additional replication slots", zap.Error(err))
+			return
+		}
+
```

Comment:
this function should also be called here: https://github.com/saranhada/stolon/blob/f436c6ca248dbc47257ca235caa91b7e543accc0/cmd/keeper/keeper.go#L1572